P2553/P2553.md

```diff
@@ -438,14 +429,44 @@ with:
   ```
 
   [1]{.pnum} *Result:*  A type `E` that is a specialization of `extents` such that
-  `E::rank() == Rank && E::rank() == E::rank_dynamic()`  is `true` and `is_same_v<E::size_type,SizeT>` is `true`.
+  `E::rank() == Rank && E::rank() == E::rank_dynamic()`  is `true` and `typename E::size_type` denotes `SizeT>.
+
+## In subsection 22.7.X.3.1 [mdspna.layout.left.ctors]
+
+* After paragraph 3 add new paragraph:
+
+*Preconditions*: `other.required_span_size()` is a representable value of `size_type` ([basic.fundamental]).
+
+* After paragraph 5 add new paragraph:
+
+*Preconditions*: `other.required_span_size()` is a representable value of `size_type` ([basic.fundamental]).
+
+* Change paragraph 8 to:
+
+*Preconditions*: 
+
+  - If `extents_type::rank() > 0` is `true` then for all $r$ in the range $[$`0, extents_type::rank()`$)$, `other.stride(r)` equals `extents().`_fwd-prod-of-extents_`(r)`, and
```

Comment:
Would you consider not putting the comma in the mathematical interval in code font?
```suggestion
  - If `extents_type::rank() > 0` is `true` then for all $r$ in the range $[$`0`, `extents_type::rank()`$)$, `other.stride(r)` equals `extents().`_fwd-prod-of-extents_`(r)`, and
```